packages/teleport/src/Apps/AddApp/useAddApp.ts

```diff
@@ -14,9 +14,10 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { useState } from 'react';
-import TeleportContext from 'teleport/teleportContext';
+import { useState, useEffect } from 'react';
 import useAttempt from 'shared/hooks/useAttemptNext';
+import { createAppBashCommand } from 'teleport/services/commands';
```

Comment:
our service layer is for containing api calls and formatting the values that return from it (the `makeXXX` funcs).
utility funcs like this, if only used in one place for now, it can be defined closest to where it is used (defined in this file)

Reply:
Thanks, I removed `packages/teleport/src/services/commands.ts` and moved `createNodeBashCommand` to the files they are used (but not inside the custom hook itself)